pyproject.toml

```
@@ -1,126 +1,127 @@
-# [build-system]
-# build-backend = "setuptools.build_meta"
-# requires = [
-# "setuptools",
-# "setuptools-scm",
-# ]
-
 [project]
+name = "crabs"
 authors = [
     {email = "n.aznan@ucl.ac.uk", name = "Nik Khadijah Nik Aznan"},
     {email = "s.minano@ucl.ac.uk", name = "Sofia Minano"},
 ]
+description = "A toolkit for detecting and tracking crabs in the field."
+readme = "README.md"
+requires-python = ">=3.9.0"
+dynamic = ["version"]
+
+license = {text = "BSD-3-Clause"}
+urls.homepage = "https://github.com/SainsburyWellcomeCentre/crabs-exploration"
+
 classifiers = [
-    "Operating System :: POSIX",
+    "Development Status :: 2 - Pre-Alpha",
+    "Programming Language :: Python",
     "Programming Language :: Python :: 3",
     "Programming Language :: Python :: 3.9",
-    "Typing :: Typed",
+    "Programming Language :: Python :: 3.10",
+    "Operating System :: OS Independent",
```

```suggestion
    "Operating System :: POSIX",
```
since we don't test Windows?

---

Yes, and we're developing stuff 100% targetted for the SWC cluster.